app/controllers/ctc/ctc_pages_controller.rb

```diff
@@ -74,7 +74,13 @@ def how_do_i_get_an_itin; end
     def completed; end
 
     def puerto_rico
-      redirect_to action: :home unless Flipper.enabled?(:puerto_rico_home_location)
+      redirect_to action: :home and return unless Flipper.enabled?(:puerto_rico_home_location)
+
+      if params[:locale].nil?
+        I18n.with_locale('es') do
+          render :puerto_rico and return
```

Comment:
Nice simple `if`!

For this line, I'd prefer `redirect_to` to change the URL to contain `/es` . I guess it doesn't matter though! So it's not a blocker.

Reply:
oh i see... you actually changed it from this.